Makefile

```diff
@@ -35,7 +35,7 @@ PKG_MANAGER ?= $(shell command -v dnf yum|head -n1)
 # ~/.local/bin is not in PATH on all systems
 PRE_COMMIT = $(shell command -v bin/venv/bin/pre-commit ~/.local/bin/pre-commit pre-commit | head -n1)
 
-SOURCES = $(shell find . -path './.*' -prune -o -name "*.go")
+SOURCES = $(shell find . -path './.*' -prune -o -name "*.go" -print)
```

Comment:
We did something similar in CRI-O as well, but excluded test files:
```
GO_FILES := $(shell find . -type f -name '*.go' -not -name '*_test.go')
```
How to get the best out of both `find` commands? :upside_down_face: